docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: docs
+title: Getting Started
+permalink: /documentation/getting-started/
+---
+# Getting Started
+
+This guide will provide you with step-by-step instructions on how to create a functional web test with TestCafe.
+You will learn how to install TestCafe, write a test, run the test on multiple browsers and view test results.
+During creating a test, you will learn the basics of TestCafe API, advancing step by step, from the simple to more elaborate testing techniques.
+
+All tests in this guide target a sample page available at [http://testcafe.devexpress.com/Example](http://testcafe.devexpress.com/Example). You can try running each of them against this page by yourself.
+
+This topic contains the following sections.
+
+* [Setting up Test Project](#setting-up-test-project)
+* [Installing TestCafe](#installing-testcafe)
+* [Writing Test](#writing-test)
+  * [General Code Structure](#general-code-structure)
+  * [Performing Actions on Page](#performing-actions-on-page)
+  * [Assertions](#assertions)
+  * [Dialog Handling](#dialog-handling)
+  * [Authentication](#authentication)
+* [Running Test](#running-test)
+* [Viewing Test Results](#viewing-test-results)
```

Comment:
Do we need this reference? We don't have anything like this on other pages. Also, I'm not sure it makes sense to put it in the page header (it should be in a menu if we really need this)